libcaf_net/caf/net/web_socket/framing.cpp

```diff
@@ -63,7 +80,7 @@ void framing::abort(const error& reason) {
   up_->abort(reason);
 }
 
-ptrdiff_t framing::consume(byte_span buffer, byte_span) {
+ptrdiff_t framing::consume(byte_span buffer, byte_span delta) {
   // Make sure we're overriding any 'exactly' setting.
   down_->configure_read(receive_policy::up_to(2048));
   // Parse header.
```

Comment:
We've switched the model from "get all bytes at once" to "incrementally parse". However, we still call `decode_header` unconditionally each time. And this function isn't trivial, so it'll add quite a bit of overhead if the client sends small chunks. We can tackle this as a followup, but I think we should address it.

Reply:
Will do. Discussion in the below comment.